.github/workflows/release.yml

```
@@ -40,6 +37,12 @@ jobs:
         uses: subosito/flutter-action@v2
         with:
           channel: stable
+          cache: true
+
+      - name: Cache Gradle
+        uses: burrunan/gradle-cache-action@v1
+        with:
+          build-root-directory: ${{ github.workspace }}/android
 
       - name: Install dependencies
         run: npm ci
```

ReVanced repository uses install instead of clean-install

```suggestion
        run: npm i
```